src/styles/main.scss

```diff
@@ -0,0 +1,103 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.product {
+  @include flex-column;
+
+  width: $product-full-width;
+  background-color: $background;
+  align-items: center;
+  text-align: center;
+  padding-block: 34px 0;
```

Comment:
you have to add paddings according to the mockup. Paddings should be used as part of card and not as margins like in your solution right now.